web/src/containers/OpenOrders/index.tsx

```diff
@@ -135,7 +135,7 @@ export class OpenOrdersContainer extends React.Component<Props> {
                 localeDate(created_at, 'fullDate'),
                 <span style={{ color: setTradeColor(side).color }} key={id}>{Decimal.format(price, priceFixed, ',')}</span>,
                 <span style={{ color: setTradeColor(side).color }} key={id}>{Decimal.format(remainingAmount, amountFixed, ',')}</span>,
-                <span style={{ color: setTradeColor(side).color }} key={id}>{Decimal.format(total, amountFixed, ',')}</span>,
+                <span style={{ color: setTradeColor(side).color }} key={id}>{Decimal.format(+remaining_volume, amountFixed, ',')}</span>,
```

Comment:
but there is still `total` [in headers](https://github.com/openware/baseapp/blob/611ca5a6429ca91eb4b55b3710d59dbd8c3ffc46/web/src/containers/OpenOrders/index.tsx#L87), why did you change value to remaining volume?